packages/mock/src/types.ts

```diff
@@ -8,6 +8,7 @@ export interface MockDefinition {
   name: string;
   overrided?: boolean;
   includedProperties?: string[];
+  functions?: string[];
```

Comment:
I would like to rename `functions` for a couple of reasons.

1. It is difficult to tell whether it is a list of typescript `Function` or a collection of function definitions as a string.
2. `MockDefinition` already has a field `value` to get the value, so we need to clarify what value this field is.

If it's functions of split mock definition, how about `splitMockImplementions`?

Reply:
Good suggestion, done!